inference-engine/src/cldnn_engine/cldnn_program.cpp

```diff
@@ -238,7 +238,8 @@ Program::Program(InferenceEngine::ICNNNetwork& network, std::shared_ptr<const cl
 
         auto transforms = LowPrecisionTransformer::getAllTransformations(params)
                 .add<FullyConnectedTransformation>(LayerTransformation::Params(params).setSupportAsymmetricQuantization(false), "FullyConnected")
-                .add<FullyConnectedTransformation>(LayerTransformation::Params(params).setSupportAsymmetricQuantization(false), "GEMM");
+                .add<FullyConnectedTransformation>(LayerTransformation::Params(params).setSupportAsymmetricQuantization(false), "GEMM")
+                .removeTransformations("Normalize");
```

Comment:
Fix for NormalizeL2 was merged to master (#720). Please remove this line.

Reply:
done, thanks!